src/components/app/details/appConfig/AppConfig.tsx

```diff
@@ -361,7 +361,7 @@ export default function AppConfig({ appName, isJobView }: AppConfigProps) {
                 .slice()
                 .reverse()
                 .find((stage) => stage.status)
-            _lastConfiguredStage = lastConfiguredStage.stageName
+            _lastConfiguredStage = lastConfiguredStage?.stageName || ''
```

Comment:
we can replicate the line no 349 check here. i.e. if(!lastConfiguredStage){

Reply:
done